SingularityUI/app/views/requestForm.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import View from './view';
+
+import { Provider } from 'react-redux';
+
+import RequestFormPage from '../components/requestForm/RequestForm'
+
+
+class RequestFormView extends View {
+  initialize({store, editing}) {
+    this.store = store;
+    this.editing = editing;
```

Comment:
can you explain why the explicit `editing` prop instead of basing the decision on if a request was loaded?